external-import/harfanglab-incidents/src/harfanglab_incidents_connector/converter_to_stix.py

```diff
@@ -337,8 +337,11 @@ def create_observables(
                         observable = self._create_domain_name(alert_intelligence)
                 case "url":
                     observable = self._create_url(alert_intelligence)
-        if isinstance(
-            alert_intelligence, (harfanglab.SigmaRule, harfanglab.YaraSignature)
+        if (
```

Comment:
Core of the PR

Reply:
@flavienSindou Thank you for the changes, tested locally:
- check behavior before changes
- check behavior after your changes

Everything works perfectly well, good for me :)